Gateway/Http/Client/ClientService.php

```diff
@@ -95,12 +95,13 @@ public function __construct(
     public function placeRequest(TransferInterface $transferObject)
     {
         $log = [];
-        $log['uri'] = $transferObject->getUri();
         $response = [];
+        $_requestUri = '';
```

Comment:
This variable definition and assignment is not necessary

Reply:
Also can we just use keywords like `private` instead of using the `_` to define the variable.